README.md

```diff
@@ -1,3 +1,7 @@
-# hello-microverse
+Hello Microverse
+This is the first project with microverse. Includes index.html & style.css files along with linter setup.
```

Comment:
- [ ] As requested by the previous code reviewer , kindly make sure to add a README file by copying the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and make sure to customize it according to your project. :wink: